### ISSUE TITLE
Content of the page is getting truncated and is not visible when resized to 400%.

### ISSUE THREAD

**Issue by tyveille** [microsoft/fluentui#18104]
### Environment Information

#URL: use the website
OS Build: Version 20H2(19042.867)
Browser: Version 89.0.4389.90 (Official Build) (64-bit)

### Describe the issue:

User with visual impairment using resize the text mode 400% will not be able to interact with the element if controls are not visible after applying the resize mode.

### Please provide a reproduction of the issue in a codepen:
 
1. Navigate to left pane and select it, navigate to "FloatingPeoplePicker ' and select it, "FloatingPeoplePicker"  screen will appear.
2. Resize the page to 400%. 
3. Observe and verify the issue.

#### Actual behavior:

On resizing the screen to 400%,content under the Navigation controller page is getting truncated such as, Heading and other controls on the page.

#### Expected behavior:

On resizing the screen to 400%,content under the Navigation controller page should not get truncated such as, Heading and other controls on the page.

### Documentation describing expected behavior

MAS 1.4.10 – Reflow

**Comment by gouttierre** [comment]
@tyveille -Thanks for filing an issue, but could you provide more details? Some screenshots and possibly the fluent version would help us out. Thanks

**Comment by tyveille** [comment]
Choose any component with a long name and zoom to 400%. You will see the header get truncated with no way to view the entire text.
![image](https://user-images.githubusercontent.com/22622621/121568192-4acb0180-c9d4-11eb-9bb0-3149345311d0.png)

**Comment by tyveille** [comment]
Any updates for this issue?
